core/src/main/java/com/linecorp/armeria/client/HttpResponseWrapper.java

```diff
@@ -286,7 +286,7 @@ private void cancelTimeoutAndLog(@Nullable Throwable cause, boolean cancel) {
         final StringBuilder logMsg = new StringBuilder(UNEXPECTED_EXCEPTION_MSG);
         final HttpRequest request = ctx.request();
         assert request != null;
-        final String authority = request.authority();
+        final String authority = ctx.authority();
```

Comment:
nit; it might be worth logging channel info instead since `ctx.authority()` can be a user-specified value

Reply:
Good point. I changed to log `ClientRequestContext` as some may want to know request information.